keymaster/include/optee_keymaster/ipc/keymaster_ipc.h

```diff
@@ -18,7 +18,7 @@
 
 // clang-format off
 
-#define KEYMASTER_PORT "com.android.trusty.keymaster"
+#define KEYMASTER_PORT "com.android.optee.keymaster"
 #define KEYMASTER_MAX_BUFFER_LENGTH 4096
```

Comment:
Is this related to `PAGE_SIZE` in `keymaster/include/optee_keymaster/ipc/optee_keymaster_ipc.h`? If yes, should this define be used instead of PAGE_SIZE?

Reply:
Well actually this macro is never used and was imported from trusty.

In trusty it's used here:
https://android.googlesource.com/platform/system/core/+/refs/tags/android-14.0.0_r15/trusty/keymaster/ipc/trusty_keymaster_ipc.cpp#140

But this part is specific to trusty (handle communication with trusty OS).

So I would say this macro can be removed